ansible/configs/base-infra/default_vars.yml

```diff
@@ -0,0 +1,135 @@
+---
+# default vars for base-aap2-infra
+# See README.adoc for full details
+
+env_type: base-rhel-infra
+
+output_dir: "/tmp/output_dir/{{ guid }}"      # Writable working scratch directory
+project_tag: "{{ env_type }}-{{ guid }}"      # This var is used to identify stack (cloudformation, azure resourcegroup, ...)
+
+cloud_provider: ec2                           # Default to AWS
+
+email: "{{ env_type }}@opentlc.com"
+guid: "{{ env_type }}-01"
+uuid: "{{ guid }}"
+
+cf_template_description: "{{ env_type }}-{{ guid }} Ansible Agnostic Deployer "
+
+# Dump inventories to output_dir - very useful as a dev -i option
+
+agnosticd_inventory_exporter_enable: true # Dump inventory in output_dir
+
+bookbag_git_repo: https://github.com/tonykay/wasm-oci-workshop.git
+
+ansible_service_account_user_name: devops
```

Comment:
Probably not applicable for a base config

Reply:
Removed